### PR TITLE
discarding printing of full config to avoid any sensitive data leak

### PR DESCRIPTION
**Pull request by kotharironak**
As part of the issue - https://github.com/hypertrace/hypertrace/issues/137, where we identified that some sensitive information is logged (password for postgres), we are going with option (a) listed in the issue. There is no need to print `config` here.

### CODE REVIEW

File: entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java [hunk 1]
```diff
@@ -14,8 +14,6 @@
   private final Config entityServiceConfig;
 
   public EntityServiceConfig(Config config) {
-
-    LOG.info(config.toString());
```
Comment (kotharironak) [inline entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java:18]: This is anyway not required as we are already logging appConfig in the base framework - https://github.com/hypertrace/service-framework/blob/d3370d2309d95bc8fe7ca7275f66f351454422a6/platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/PlatformService.java#L97

**Review by rish691 — APPROVED** [review]

File: entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java (hunk 1 above)
Comment (buchi-busireddy) [reply entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java:18]: Won't the credentials be printed in the app config then?

Comment (kotharironak) [reply entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java:18]: Yes. And as discussed here - https://github.com/hypertrace/hypertrace/issues/137, I am going to apply option (a) even at the framework level too. The consumer should print requires things as per need with a meaningful message to make it actionable.  So, this is anyway needed.

Comment (kotharironak) [reply entity-service/src/main/java/org/hypertrace/entity/service/EntityServiceConfig.java:18]: This is the PR for the same - https://github.com/hypertrace/service-framework/pull/22, once that is merged, I have to upgrade appropriate services, at least attribute, entity and config-bootstrapper for now for postgres sensitive information.

**Review by JBAhire — APPROVED** [review]

**Review by tim-mwangi — APPROVED** [review]

**Pull request merged**